src/components/Loading.jsx

```diff
@@ -0,0 +1,7 @@
+export default function Loading() {
+  return (
+    <div className="loading">
+      <h1>Loading...</h1>
```

Comment:
h1 might not be best here particularly if your page already has a h1, consider how this could be confusing for people using screen readers